dtde/spark/cli/cluster.py

```diff
@@ -32,15 +36,22 @@ def setup_parser(parser: argparse.ArgumentParser):
         ClusterAction.get, help="Get information about a cluster")
     list_parser = subparsers.add_parser(
         ClusterAction.list, help="List clusters in your account")
+    logs_parser = subparsers.add_parser(
+        "logs", help="Action on an app")
     ssh_parser = subparsers.add_parser(
         ClusterAction.ssh, help="SSH into the master node of a cluster")
+    submit_parser = subparsers.add_parser(
+        "submit", help="Submit a new spark job")
```

Comment:
since we want to be explicit about this being "interactive mode" perhaps update this to "Submit a new spark job to a cluster", rather an in serverless mode where the user should not care about a cluster.